packages/server/database/migrations/20201130124200-removeSlackUserAccessToken.ts

```diff
@@ -0,0 +1,23 @@
+import {R} from 'rethinkdb-ts'
+export const up = async function(r: R) {
```

Comment:
these migrations need to live in another PR otherwise if we called messageAllSlackUsers, we wouldn't message anyone.